amity/__init__.py

```diff
@@ -0,0 +1 @@
+from amity import Amity
```

Comment:
**Code Quality - Nitpick:** It's not best practice adding this in `__init__.py`.